ui/introduction/Introduction.tsx

```diff
@@ -6,7 +6,15 @@ import { redirect } from 'next/navigation'
 import { TextImage } from 'ui'
 import { useLocalizedRoutes, usePathData } from 'hooks'
 
-export default function Introduction({ children, lang }) {
+export default function Introduction({
+  children,
+  lang,
+  imagePosition,
+}: {
+  children: any
```

Comment:
we could use ReactNode as a children type instead of any?
also what types are lang?

Reply:
all langs are strings, but they're everywhere so we can address that in another PR, made the changes here though